packages/patternfly-3/patternfly-react/package.json

```diff
@@ -48,8 +48,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.6.0",
-    "react": "^16.3.1",
-    "react-dom": "^15.6.2 || ^16.2.0"
+    "react": "^16.8.1",
```

Comment:
as its peer Dependencies, I think this implies every patternfly user need to upgrade react too....not sure if we are ready for it?

Reply:
thanks, removed all peer Dependencies changes